src/trace_processor/perfetto_sql/stdlib/wattson/tasks/threads_w_processes.sql

```diff
@@ -38,3 +40,120 @@ LEFT JOIN android_process_metadata AS package
   USING (upid)
 WHERE
   dur > 0;
+
+-- Gets all IRQs and labels each slice with hard or soft IRQ
+CREATE PERFETTO TABLE _all_irq_slices AS
+SELECT
+  id,
+  ts,
+  dur,
+  parent_id,
+  extract_arg(arg_set_id, 'vec') IS NULL AS is_hard_irq
```

Comment:
This is a very inefficient way of doing this. If you do a union all of the two track types, it will be signifcantlyt more efficient.